src/week05/scripts/mint.py

```diff
@@ -56,52 +57,27 @@ def main(
     signatures = []
     if script == "nft":
         # Build script
-        save_path = assets_dir.joinpath(f"nft_{token_name}")
         script_path = lecture_dir.joinpath("nft.py")
         oref = TxOutRef(
             id=TxId(bytes(utxo_to_spend.input.transaction_id)),
             idx=utxo_to_spend.input.index,
         )
         tn_bytes = bytes(token_name, encoding="utf-8")
-        tn_json = json.dumps({"bytes": tn_bytes.hex()})
-        subprocess.run(
-            [
-                "opshin",
-                "-o",
-                str(save_path),
-                "build",
-                str(script_path),
-                oref.to_json(),
-                tn_json,
-            ],
-            check=True,
-        )
-        cbor_path = save_path.joinpath("script.cbor")
+        script_built = build(script_path, oref, tn_bytes)
+        cbor_hex = script_built.cbor_hex
     elif script == "signed":
         # Build script
-        save_path = assets_dir.joinpath(f"signed_{wallet_name}")
         script_path = lecture_dir.joinpath("signed.py")
         pkh = bytes(get_address(wallet_name).payment_part)
         signatures.append(pkh)
-        pkh_json = json.dumps({"bytes": pkh.hex()})
-        subprocess.run(
-            [
-                "opshin",
-                "-o",
-                str(save_path),
-                "build",
-                str(script_path),
-                pkh_json,
-            ],
-            check=True,
-        )
-        cbor_path = save_path.joinpath("script.cbor")
+        script_built = build(script_path, pkh)
```

Comment:
Looks like this part is erroring because it expects plutus data but our script expects a bytes argument:

```
  File "...\opshin-pioneer-program\src\week05\scripts\mint.py", line 73, in main
    script_built = build(script_path, pkh)
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "...\opshin-pioneer-program-8MnTygC3-py3.11\Lib\site-packages\opshin\builder.py", line 43, in build
    code = uplc.ast.Apply(code, uplc.ast.data_from_cbor(d.to_cbor("bytes")))
                                                        ^^^^^^^^^
AttributeError: 'bytes' object has no attribute 'to_cbor'
```